frontend/src/components/HistoryPage.js

```diff
@@ -0,0 +1,70 @@
+import { useState, useEffect } from "react"
+import { useNavigate } from "react-router-dom"
+import NavBar from './NavBar'
+import { Table } from 'react-bootstrap'
+import userService from "../services/userService.js"
+import HistoryService from "../services/historyService.js"
+
+const HistoryPage = () => {
+    const [user, setUser] = useState("");
+    const [histories, setHistories] = useState([])
+
+    const navigate = useNavigate()
+
+    useEffect( () => {
+        initializePage();
+    }, [])
+
+    const initializePage = async () => {
+        try {
+            const token = sessionStorage.getItem("jwt");
+            const res = await userService.getUser(token);
```

Comment:
Recommend to change `res` to `user` to be more concise.

Reply:
Used res most probably because there is a setState variable with the name user so might be confusing or cause bugs